package.json

```diff
@@ -10,7 +10,7 @@
     "dev": "nodemon",
     "postinstall": "npm rebuild node-sass",
     "test": "cross-env NODE_ENV=test jest",
-    "test-watch": "yarn test --watch"
+    "test-watch": "yarn test --watchAll"
```

Comment:
I love that you can still do this.

We should make sure that develop is locked down from this.

(note that the merge conflict was resolved by GH pushing to develop then merging into the work branch)